third_party/ijar/BUILD

```diff
@@ -94,6 +94,12 @@ filegroup(
     visibility = ["//third_party:__pkg__"],
 )
 
+filegroup(
```

Comment:
nit: do we need the `srcs` filegroup then?

Reply:
Yes, it's used to build the dist archive. Why shouldn't we need it?